lib/default-options.js

```diff
@@ -16,7 +16,8 @@ module.exports = function defaultOptions () {
             'assert.notDeepEqual(actual, expected, [message])',
             'assert.deepStrictEqual(actual, expected, [message])',
             'assert.notDeepStrictEqual(actual, expected, [message])'
-        ]
+        ],
+        additionalMethods: []
```

Comment:
Hmm...It's hard..
If I understand this correctly, this property is to list assertion method patterns that are not enhanced by instrumentation phase but want to track calls at runtime. Right?